src/main/java/org/dominokit/domino/ui/forms/Select.java

```diff
@@ -62,7 +62,8 @@ public Select() {
         initListeners();
         buttonElement
                 .appendChild(buttonValueContainer)
-                .appendChild(iconContainer);
+                .appendChild(iconContainer)
+                .asElement().type = "button";        
```

Comment:
I would prefer to use the builder to set the attribute in the initialization statement 
https://github.com/DominoKit/domino-ui/blob/master/src/main/java/org/dominokit/domino/ui/forms/Select.java#L41